packages/block-library/src/comment-date/index.php

```diff
@@ -31,14 +31,14 @@ function render_block_core_comment_date( $attributes, $content, $block ) {
 	$link               = get_comment_link( $comment );
 
 	if ( ! empty( $attributes['isLink'] ) ) {
-		$formatted_date = sprintf( '<a href="%1s">%2s</a>', $link, $formatted_date );
+		$formatted_date = sprintf( '<a href="%1s">%2s</a>', esc_url( $link ), esc_html( $formatted_date ) );
```

Comment:
`get_comment_date()` is never escaped anywhere else in core it seems, so I think it's safe to omit the `esc_html()` here

Reply:
I wonder _why_ though. There is no escaping going on within that function. As guess as it's not user generated content in the sense that it computes a date string from a generated post date and an option.

My reason for escaping (as always) is that you shouldn't trust any variable to remain "as is" when outputting HTML.

Reply:
Ah thanks for clarifying. My tired afternoon brain missed this. I've reverted the change.